frontend/src/routes/ReceiptForm/FormStructure.js

```diff
@@ -0,0 +1,213 @@
+        <RecievedApplicationBanner>
+          <span>Vi har mottatt søknaden din!</span>
+          <Icon name="checkmark" size="1.8rem" />
+        </RecievedApplicationBanner>
+        <TimeStamp>
+          <Icon name="time" />
+          Søknaden ble sist registrert
+          {myApplication && (
+            <StyledSpan bold>
+              <Moment format="dddd Do MMMM, \k\l. HH:mm:ss">
+                {myApplication.updated_at}
+              </Moment>
+            </StyledSpan>
+          )}
+        </TimeStamp>
+        <EditWrapper>
+          <EditInfo>
+            <Text>
+              Du kan
+              <StyledSpan bold> fritt endre søknaden</StyledSpan> din frem til{" "}
+              {admission && (
+                <StyledSpan bold red>
+                  <Moment format="dddd Do MMMM">
+                    {admission.public_deadline}
+                  </Moment>
+                </StyledSpan>
+              )}{" "}
+              og komiteene vil kun se den siste versjonen.
+            </Text>
+            <Notice>
+              <StyledSpan bold>Merk:</StyledSpan> Oppdateringer etter
+              søknadsfristen kan ikke garanteres å bli sett av komiteen(e) du
+              søker deg til.
+            </Notice>
+          </EditInfo>
+          <EditActions>
+            <LegoButton
+              icon="arrow-forward"
+              iconPrefix="ios"
+              onClick={toggleIsEditing}
+            >
+              Endre søknad
+            </LegoButton>
+            <ConfirmModal
+              title="Slett søknad"
+              Component={({ onClick }) => (
+                <LegoButton
+                  onClick={onClick}
+                  buttonStyle="secondary"
+                  size="medium"
+                >
+                  {deleteApplicationMutation.isLoading
+                    ? "Sletter søknad..."
+                    : deleteApplicationMutation.isError
+                    ? "Klarte ikke slette søknad"
+                    : deleteApplicationMutation.isSuccess
+                    ? "Søknad slettet!"
+                    : "Slett søknad"}
+                </LegoButton>
+              )}
+              message="Er du sikker på at du vil slette søknaden din?"
+              onConfirm={() =>
+                deleteApplicationMutation.mutate(null, {
+                  onSuccess: () => {
+                    clearAllDrafts();
+                    navigate("/");
+                  },
+                })
+              }
+            />
+          </EditActions>
+        </EditWrapper>
+      </RecieptInfo>
+
+      <FormHeader>
+        <Title>Innsendt data</Title>
+      </FormHeader>
+      <Form>
+        <SeparatorLine />
+        <GeneralInfoSection>
+          <SectionHeader>Generelt</SectionHeader>
+          <HelpText>
+            <Icon name="information-circle-outline" />
+            Mobilnummeret vil bli brukt til å kalle deg inn på intervju av
+            komitéledere.
```

Comment:
Should we add some info of how the phone number (and the "Innsendt data" in general) is treated during and after the admissions are over? Nevertheless out of scope of what you're trying to achieve with this PR.